src/shared/components/NavigationMenu/Home.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import RecentlyVisited from '../RecentlyVisited';
+import './nav-menu.less';
+import { Button, Divider } from 'antd';
+
+export interface NavMenuPageProps {
+  path: string;
+  goTo: (path: string) => void;
+}
+
+interface NavMenuHomeProps extends NavMenuPageProps {
+  goToProject(orgLabel: string, projectLabel: string): void;
+}
+
+export const NavMenuHome: React.FunctionComponent<NavMenuHomeProps> = props => {
+  const { path, goTo, goToProject } = props;
+  return (
+    <div className="page -home">
+      <p>
+        Projects are where you can compartmentalize your data and access
```

Comment:
I think for label like that we should maybe try to come up with something that matches the documentation a little bit more: https://bluebrainnexus.io/docs/api/1.1/admin/admin-projects-api.html

Reply:
The verbiage in the documentation is way too low level for a description though

Reply:
Agreed, but we what I mean is maybe re-use some of the vocabulary, like:
"Projects belong to an organisation and allow you to group and categorise resources and provide isolation from other projects" or whatever. I think it might be worth asking on slack in order to get everyone's approval, rather that coming up with something ourselves. Or you just don't say anything for now.